pkg/deploy/api/types.go

```diff
@@ -405,6 +405,10 @@ type DeploymentConfigList struct {
 // DeploymentConfigRollback provides the input to rollback generation.
 type DeploymentConfigRollback struct {
 	unversioned.TypeMeta
+	// Name of the deployment config that will be rolled back.
+	Name string
+	// UpdatedAnnotations is a set of new annotations that will be added in the deployment config.
+	UpdatedAnnotations map[string]string
```

Comment:
For adding/updating annotations of the deployment config during rollback.
It's not exposed in the cli yet upstream but I would guess it's meant to be
used at some point (a flag on `rollout undo`?)
https://github.com/kubernetes/kubernetes/blob/1d128614bc463752ce0233fcee11d8bb9778963e/pkg/apis/extensions/v1beta1/types.go#L263-L266

On Tue, Jun 7, 2016 at 5:35 PM, Clayton Coleman <notifications@github.com>
wrote:

> In pkg/deploy/api/types.go
> <https://github.com/openshift/origin/pull/9135#discussion_r66095178>:
>
> > @@ -405,6 +405,10 @@ type DeploymentConfigList struct {
> >  // DeploymentConfigRollback provides the input to rollback generation.
> >  type DeploymentConfigRollback struct {
> >  	unversioned.TypeMeta
> > +	// Name of the deployment config that will be rolled back.
> > +	Name string
> > +	// UpdatedAnnotations is a set of new annotations that will be added in the deployment config.
> > +	UpdatedAnnotations map[string]string
>
> What is this for?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9135/files/0a31c175b6640862d4e1303a260629570af5f236#r66095178>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADuFf92tPs3VUWCInViWC2EH18lg8zsZks5qJY_OgaJpZM4IsjxM>
> .
>